app/Http/Controllers/Admin/ChoiceController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Word;
+use App\Choice;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\ChoiceRequest;
+
+class ChoiceController extends Controller
+{
+    public function list(Word $word)
+    {
+        $choices = Choice::where('word_id', $word->id)
+            ->orderBy('id', 'asc')
+            ->paginate(10);
```

Comment:
You can actually just directly say like this:

```
    public function list(Word $word)
    {
        $choices = $word->choices()->orderBy('id', 'asc')->paginate(10);
```

Reply:
this can be applied to those model who have relationship

Reply:
I changed list function to use relationship.